src/libnnstreamer-edge/nnstreamer-edge-internal.c

```diff
@@ -962,6 +965,39 @@ nns_edge_create_handle (const char *id, const char *topic, nns_edge_h * edge_h)
   return NNS_EDGE_ERROR_NONE;
 }
 
+/**
+ * @brief Get IP address
+ */
+static gchar *
+_get_ip_address (void)
```

Comment:
suggestion: move this to common as an util function.

Reply:
Moved. Thanks.